ambry-api/src/main/java/com/github/ambry/protocol/RequestAPI.java

```diff
@@ -55,6 +55,15 @@ public interface RequestAPI {
    */
   void handleDeleteRequest(NetworkRequest request) throws IOException, InterruptedException;
 
+  /**
+   * Deletes a list of blobs from the store.
```

Comment:
Can you add more info in your comment about what this api does, like handling partial complete and full delete scenario.